### PR TITLE
[3532][16.0][IMP] product_plm_import: add purchase_description_rule

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3532](https://www.quartile.co/web#id=3532&cids=3&model=project.task&view_type=form)

### CODE REVIEW

File: product_plm_import/models/plm_product_mapping.py [hunk 1]
```diff
@@ -21,6 +21,13 @@ class PlmProductMapping(models.Model):
         .selection,
         required=True,
     )
+    purchase_description_rule = fields.Selection(
+        selection=[
+            ("standard", "Standard Description"),
+            ("plm", "PLM Description"),
```
Comment (yostashiro) [inline product_plm_import/models/plm_product_mapping.py]: To be more descriptive.
```suggestion
            ("standard", "Standard: [Generic Name] / [Drawing] / [Spec]"),
            ("generic_name", "Generic Name),
```

**Review by yostashiro — COMMENTED** [review]
@rinaldifirdaus Please follow up to update the translation and readme - I realize that there is currently no explanation on PLM-product mapping.  Please add a brief explanation on this in CONFIGURE.rst.

**Comment by rinaldifirdaus** [comment]
>Please follow up to update the translation and readme - I realize that there is currently no explanation on PLM-product mapping. Please add a brief explanation on this in CONFIGURE.rst.

@yostashiro san, i've added the PLM-product mapping on configure file on the new commit.
regarding translations, i've just added the new term that improved on this PR. 
there are many term that is not translated yet, i remember that client inform us that message should be in english. is the untranslated term based on this client request or all of this term should be translated? 

Thank you

**Review by yostashiro — APPROVED** [review]
Looks good!

**Pull request merged**